include/dynamic_graph_manager/tools.hpp

```diff
@@ -7,13 +7,22 @@
  * @date 2019-05-22
  */
 
-#include <dynamic-graph/linear-algebra.h>
-
 #include "yaml_utils/yaml_cpp_fwd.hpp"
 
 #ifndef TOOLS_HH
 #define TOOLS_HH
 
+#pragma GCC diagnostic push
```

Comment:
dangerous for real-time?

Reply:
Why do you think this line is dangerous for real-time? I literately copied the code from `dynamic-graph/linear-algebra.h` [1] and they use the same analysis. So if there is a problem with real-time compatibility, we would have it already right now.

[1]: http://docs.ros.org/en/melodic/api/dynamic-graph/html/linear-algebra_8h_source.html